sonar-plugin-api/src/main/java/org/sonar/api/rules/ActiveRule.java

```diff
@@ -36,6 +36,7 @@
   private Integer id;
   private Rule rule;
   private RulePriority severity;
+  private RulePriority overriddenSeverity;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "RulePriority"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)